docs/administration/audit-logs.md

```diff
@@ -0,0 +1,41 @@
+# Audit Logs
+
+Audit logs are a record of all user actions performed on the platform across experiments, feature flags, metrics, and facts.
```

Comment:
just for reference and completeness, here's the full list of models we track with audit logs, in case you want to be more specific
```
  Allocation,
  Experiment,
  ExperimentMetricGroup,
  ExperimentMetricGroupMetric,
  ExperimentVariation,
  ExposureSchedule,
  FunnelMetricAggregation,
  Metric,
  MetricCollectionMetric,
  MetricEventAggregation,
  MetricEventEntity,
  MetricEventMeasure,
  MetricEventSource,
  PercentileMetricAggregation,
  VariationWeight,
```